docs/whats-new.md

```diff
@@ -40,13 +56,15 @@ For custom layer authors: the attribute system is simplified. One may now use `t
 ### Minor Improvements and Bug Fixes
 
 - `OrbitController` now supports 360 degree rotation on both axis. Relax `minRotationX` and `maxRotationX` to use this feature.
+- Built-in tooltip: A new prop [getTooltip](/docs/api-reference/deck.md#gettooltip-function-optional) is added to the `Deck` class. By supplying this callback, an app may specify the content and styling of a built-in tooltip.
```

Comment:
Auto-tooltip seems like a new feature and worth its own entry at the end of the list above.